ironfish-rust/src/transaction/mod.rs

```diff
@@ -204,6 +190,7 @@ impl ProposedTransaction {
     /// aka: self.value_balance - intended_transaction_fee - change = 0
     pub fn post(
         &mut self,
+        spender_key: SaplingKey,
```

Comment:
Wondering if this should be a `&SaplingKey`, to avoid excessive copying to the stack

Reply:
I'll update!